zenodo/base/format_templates/DataCite.xsl

```diff
@@ -124,6 +140,24 @@ exclude-result-prefixes="marc fn dc invenio">
                         <nameIdentifier nameIdentifierScheme="info">info:eu-repo/grantAgreement/EC/FP7/<xsl:value-of select="subfield[@code='c']"/></nameIdentifier>
                     </contributor>
                 </xsl:for-each>
+                <xsl:for-each select="datafield[@tag=700][subfield[@code='4']='ths']">
```

Comment:
Is it okay to  added `Supervisors` to `DataCite`? (It was in `DataCite3` but not `DataCite`)?